### PR TITLE
Load environment dynamically

### PR DESCRIPTION
**Pull request by d3473r**

### CODE REVIEW

File: frontend/src/assets/config.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (makoeta) [inline frontend/src/assets/config.json:1]: Remove the file from the repo?

**Review by makoeta — CHANGES_REQUESTED** [review]

File: frontend/libs/game/src/lib/game/game.component.ts [hunk 2]
```diff
@@ -80,7 +80,7 @@ export class GameComponent implements OnInit {
     );
   }
 
-  onKey(event: any) {
-    const inputValue = event.target.value;
+  onKey(_event: KeyboardEvent) {
+    // do nothing
```
[the remark this answers is not in the document]
Comment (d3473r) [reply frontend/libs/game/src/lib/game/game.component.ts:84]: Yeah, i don't know what it does. We should use reactive forms either way to handle user input

File: frontend/src/assets/config.json (hunk 1 above)
Comment (d3473r) [reply frontend/src/assets/config.json:1]: No, this is the default config for running locally. We'll overwrite this file with a mount for deployment

**Review by makoeta — APPROVED** [review]

**Comment by makoeta** [comment]
@d3473r lgtm

**Pull request merged**